### ISSUE TITLE
commonjsGlobal.describe is not a function

### ISSUE THREAD

**Issue by dkniffin** [stalniy/bdd-lazy-var#42]
I'm trying to use the following code:

```
describe('utils/canDoAThing', function() {
  subject(() => canDoAThing(get.user, get.tour));
  ...

  sharedExamplesFor('a disallowed action', () => {
    it('returns false', function() {
      expect(get.subject).to.be.false;
    });
  });

  ...

  itBehavesLike('a disallowed action');
});
```

And I get this error:

```
  commonjsGlobal.describe('behaves like ' + name, function () {
                 ^

TypeError: commonjsGlobal.describe is not a function
    at itBehavesLike (/Users/dkniffin/Repositories/project/node_modules/bdd-lazy-var/getter.js:343:18)
```

**Comment by stalniy** [comment]
Strange. I will check it tomorrow. Thanks for the issue!

**Comment by stalniy** [comment]
this is how rollup tries to detect `global` object:

```js
var commonjsGlobal = typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};
```

As far as I remember you add `window` mock to `global` manually. That's why you get this error. I will investigate whether there are other options for rollup to detect `global` object

**Comment by stalniy** [comment]
Found solution it's possible to utilize `rollup-plugin-inject` to change check order. I will check `global`, `self` and only then `window`. This will allow to mock `window` in node envs.

Also just a note about your code:

`sharedExamplesFor` defines reusable tests globally, they are not scoped to `describe` where you define it. That means there is no reason to put it into `describe` blocks

**Comment by stalniy** [comment]
Published 2.2.3 with update `global` declarations. Please try it and let me know whether everything works ok

**Comment by dkniffin** [comment]
💥 Yep, that worked! Thanks so much for helping me for all your help in getting this library working in my environment.
